### ISSUE TITLE
Troubleshoot 2.5k muted users npub bug

### ISSUE THREAD

**Issue by alltheseas** [vitorpamplona/amethyst#1153]
https://njump.me/nevent1qqs25r9sz0z65cf28tekgznu9e0gn6g5e0h3390300c38hznac7473gprfmhxue69uhhyetvv9ujummjv9hxwetsd9kxctnyv4mqzxmhwden5te0wejhycnfwf5kx6rp9ehx7um5wgcjucm0d5q3wamnwvaz7tmxwfjkuuewdehhxarjxgcjucm0d5q3camnwvaz7tmwdaehgu3wvf5hgcm0d9hx2u3wwdhkx6tpdsancl68

**Comment by vitorpamplona** [comment]
Yeah, we need to stop trying to render the complete list in the post and open a new route to view this with Lazy Loading. 

Though, very few relays accept lists this big, so it's surprising that this is even downloading.

**Comment by KotlinGeekDev** [comment]
Hmm @vitorpamplona this issue seems not relevant anymore?
